### PR TITLE
Bug fix.

### PR DESCRIPTION
**Pull request by romacher**

### CODE REVIEW

File: src/draggable.js [hunk 1]
```diff
@@ -9,18 +9,20 @@ export default class Draggable {
     }
 
     onMouseDown(e) {
-        if (!this.start) {
-            this.start = e;
-        }
+        this.start = e;
+
+        const oldTransform= this.item.element.style.transform.replace(/[(px,)]/g, ' ').split(/\s+/g)
```
Comment (humanamburu) [inline src/draggable.js]: I reckon you can store the previous position as part of `Card` or `Item`

File: src/draggable.js [hunk 2]
```diff
@@ -9,18 +9,20 @@ export default class Draggable {
...
+        this.oldX = +oldTransform[1]
```
Comment (humanamburu) [inline src/draggable.js]: also please don't use `old` adjective, it sounds more naturally like `previous`

**Review by humanamburu — CHANGES_REQUESTED** [review]

File: src/item.js [hunk 3]
```diff
@@ -18,6 +20,12 @@ export default class Item {
         this.element.removeEventListener(event, fn);
     }
 
+    setPreviousPosition() {
+        const oldTransform= this.element.style.transform.replace(/[(px,)]/g, ' ').split(/\s+/g)
```
Comment (humanamburu) [inline src/item.js]: I'm 100% sure that you can store coordinates in the class and avoid string parsing by regexp. It's an antipattern to use regexp for parsing styles and code.

File: src/draggable.js [hunk 4]
```diff
@@ -9,18 +9,16 @@ export default class Draggable {
...
         document.addEventListener('mouseup', this.onMouseUp);
     }
 
-    onMouseMove({ clientX, clientY }) {
+    onMouseMove(e) {
         this.item.setPosition(
-            clientX - this.start.clientX,
-            clientY - this.start.clientY,
+            this.item.previousX ? this.item.previousX + e.clientX - this.start.clientX : e.clientX - this.start.clientX,
+            this.item.previousY ? this.item.previousY + e.clientY - this.start.clientY : e.clientY - this.start.clientY
```
Comment (humanamburu) [inline src/draggable.js]: I guess u can also simplify this to split calculations and ternary operator

**Review by humanamburu — CHANGES_REQUESTED** [review]

File: src/item.js (hunk 3 above)
Comment (romacher) [reply src/item.js]: fixed

File: src/draggable.js (hunk 4 above)
Comment (romacher) [reply src/draggable.js]: fixed

**Review by humanamburu — APPROVED** [review]

**Pull request merged**